src/components/JoinUs/Work.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react'
+import breakpoint from 'styled-components-breakpoint'
+import remcalc from 'remcalc'
+import { Link } from 'gatsby'
+import { Padding } from 'styled-components-spacing'
+import styled from 'styled-components'
+import { Row, CompensatedCol, CompensatedRow } from '../grid'
+import { H2, H3, Paragraph } from '../Typography'
+import colorLuminance from 'color-luminance'
+
+import {
+  Section,
+  FirstColumn,
+  SecondColumn,
+  Separator,
+  List,
+  TitleAndBody
+} from './elements'
+
+const MAX_CASE_STUDIES = 3
+
+/* Utility function to convert an HEX color into the RGB format */
+const hexToRgb = hex => {
+  var result = /^#?([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})$/i.exec(hex)
+  return result
+    ? {
+        r: parseInt(result[1], 16),
+        g: parseInt(result[2], 16),
+        b: parseInt(result[3], 16)
+      }
+    : null
+}
+
+/* Returns the luminance (in a 0-255 range) of a color in the HEX format */
+const getColorLuminance = hexColor => {
+  const rgbValue = hexToRgb(`#${hexColor}`)
+
+  return colorLuminance(rgbValue.r, rgbValue.g, rgbValue.b)
+}
+
+const CaseStudyLink = styled(Link)`
+  background: ${props => props.bg};
+  color: ${props => props.color || props.theme.colors.white};
+  flex: 1;
+`
+
+const CaseStudy = ({ children, ...props }) => (
+  <CaseStudyLink {...props}>
+    <Padding
+      vertical={{ smallPhone: 2, smallTablet: 1.5, tablet: 3 }}
+      horizontal={{ smallPhone: 2, smallTablet: 1.5, tablet: 2 }}
+    >
+      {children}
+    </Padding>
+  </CaseStudyLink>
+)
+
+const CaseStudyWrapper = styled(Padding)`
+  display: flex;
+  width: 100%;
+`
+const CaseStudyTitle = styled(H3)`
+  font-size: ${remcalc(21)};
+  line-height: ${remcalc(24)};
+  color: ${props =>
+    props.reverse ? props.theme.colors.white : props.theme.colors.text};
+
+  ${breakpoint('largePhone')`
+    font-size: ${remcalc(17)};
+    line-height: ${remcalc(21)};
+  `};
+
+  ${breakpoint('smallTablet')`
+    font-size: ${remcalc(21)};
+    line-height: ${remcalc(24)};
+  `};
+
+  ${breakpoint('tablet')`
+    font-size: ${remcalc(26)};
+    line-height: ${remcalc(30)};
+  `};
+`
+
+const Work = ({ data: { title, list, text, someWork } }) => (
+  <Section>
+    <Row>
+      <FirstColumn>
+        <H2>{title}</H2>
+      </FirstColumn>
+      <SecondColumn>
+        <List list={list} />
+      </SecondColumn>
+    </Row>
+    <Separator />
+    <TitleAndBody title="Some of our work" body={text} />
```

Comment:
Does Greta not want to be able to edit these titles in Contentful?

Reply:
I really don't know. 
When talking with Sara, I understood that these main titles — if provided and approved by Fin/Greta — wouldn't change. 
Does every text field need to be customizable?

Reply:
Yeah I get where you're coming from. This does seem to be the existing pattern across a lot of our current pages. But the first time Sara & Greta were going through how to use Contentful, I remember her wanting to change the title for the Events section on the homepage & couldn't because it was hard coded.

Just went over to ask her in person. She said it would ideally be editable.

In an ideal world, I reckon these steps would be done before a ticket was considered 'ready' for engineering:
![image](https://user-images.githubusercontent.com/15656538/51490963-d3658a80-1da4-11e9-8769-970be628fdea.png)

whereby engineering & product would liase closely, when creating the content model for new pages.

But I doubt that was possible in the past, while the focus was on shipping as fast as possible & the team had really tight deadlines, to get things done before [DSL](https://www.designsystemslondon.com/)